examples/docker-compose/docker-compose.yaml

```diff
@@ -40,6 +40,7 @@ services:
     restart: always
     command:
       - filesystem
+      - --prometheus-url=http://prometheus:9090
```

Comment:
This is the default value. I don't think we need it set explicitly here.

Reply:
as code the default value is `http://localhost:9090` rather than `http://prometheus:9090`

Reply:
Oh. You're totally right. 😅👍